lib/ping_pong.ex

```diff
@@ -0,0 +1,38 @@
+defmodule PingPong do
+  @moduledoc """
+  Module is serving for testing connection between two elixir nodes
+  """
+
+  @doc """
+  Send message to the first node from Node.list() output.
```

Comment:
Why to _the first one_? Send to all of them or the one chosen by the caller. The first is more like a random one.